use/install.rst

```diff
@@ -73,48 +73,46 @@ If you do not have root access, skip this section.
 If you are logged in as root, you can remove ``sudo`` from the install
 commands.
 
-First, install Limnoria's optional dependencies (you can skip this
-step, but some features won't be available)::
-
-    sudo python3 -m pip install -r https://raw.githubusercontent.com/progval/Limnoria/master/requirements.txt --upgrade
+First, you need to setup a virtualenv, which is the directory where we
+will install Limnoria's code (but not its configuration)::
 
-Then Limnoria itself::
+    sudo python3 -m venv /opt/venvs/limnoria  # creates a virtualenv at the given path
```

Comment:
Running venv as root feels a bit strange. How often do people actually need virtualenvs shared between user?

It might be safer to create a dir owned by a non-root user inside /opt/venvs, and then do the pip install without root. This should create binaries that are world readable / executable anyways

Reply:
Yeah I'm starting to think we should remove this altogether